app.go

```diff
@@ -1024,7 +1033,66 @@ func (app *App) startupProcess() *App {
 	}
 
 	app.mutex.Lock()
+	defer app.mutex.Unlock()
+
+	// add routes of sub-apps
+	if !app.subAppsRoutesAdded {
+		app.appendSubAppLists(app.appList)
+		app.addSubAppsRoutes(app.appList)
```

Comment:
who adds the stack when no name is added to a route? i miss this in a central location